transforms/universal/html2parquet/python/src/html2parquet_transform.py

```diff
@@ -0,0 +1,84 @@
+import time
+from argparse import ArgumentParser, Namespace
+from typing import Any
+import zipfile
+import io
+import trafilatura
+from datetime import datetime
+
+
+import pyarrow as pa
+
+# disabled for now
+# from data_processing_ray.runtime.ray import RayTransformLauncher
+# from data_processing_ray.runtime.ray.runtime_configuration import (
+#   RayTransformRuntimeConfiguration,
+# )
+# import data_processing
+
+
+from data_processing.transform import AbstractBinaryTransform, TransformConfiguration
+from data_processing.utils import CLIArgumentProvider, get_logger, TransformUtils
+
+
+class HtmlToParquetTransform(AbstractBinaryTransform):
```

Comment:
For consistency, I would rename this and others from HtmlToParquet to Html2Parquet.